src/azure-cli/azure/cli/command_modules/backup/custom_afs.py

```diff
@@ -210,14 +210,23 @@ def restore_AzureFileShare(cmd, client, resource_group_name, vault_name, rp_name
 
 
 def list_recovery_points(cmd, client, resource_group_name, vault_name, item, start_date=None, end_date=None,
-                         use_secondary_region=None):
+                         use_secondary_region=None, is_ready_for_move=None, target_tier=None, tier=None,
+                         recommended_for_archive=None):
     if use_secondary_region:
         raise InvalidArgumentValueError(
```

Comment:
Please modify the original incorrect parts to keep the usage of the error type unified
```suggestion
        raise ArgumentUsageError(
```